spec/factories/comments.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :comment do
+    body Faker::Lorem.paragraph
+    user
```

Comment:
@valentina91p I understand from [factory_girl/GETTING_STARTED.md](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#associations) that Factory Girl understands and generates associations just by using the name of the factory to which the object generated by the current factory belongs to.

When the user a comment belongs to is involved in a test I create a different user and override the user_id attribute of the comment so the tests make sense just by reading the spec. I don't need to know the exact user that this factory is generating, but it ensures that the comment is valid when I'm testing it on its own. 

Do you want me to change this anyway?